### ISSUE TITLE
Vault repacks after leaving inspector

### ISSUE THREAD

**Issue by dhowe** [dhowe/AdNauseam#641]
should return to previously configure pack

to recreate:

1. open vault with ads
1.  drag and zoom to desired settings
1.  click ad to inspect
1.  click background to leave inspector
1.  notice that vault repacks and all drag/zoom settings are lost

**Comment by leoneckert** [comment]
oh I know why this is. Has to do with the fix of the resizing-in-vault issue. Looking into it.

**Comment by leoneckert** [comment]
So the reason why this happened comes from when I fixed the black overlay issue when resizing the window in inspection mode.
When working on that I noticed that the length of the slider didn't update when resizing in inspection mode, leaving it too long or short respectively:
<img width="1327" alt="screen shot 2016-11-23 at 13 42 07" src="https://cloud.githubusercontent.com/assets/14130931/20574656/9ea882c8-b183-11e6-8bbf-b4c9a4d07b27.png">

I thought I fixed it by calling `createSlider(true);` after closing the inspection, but it seems like the right thing to do would have been `createSlider(false);` which creates/redraws the slider but doesn't do the whole relayout. 
This should now solve both things: adjust length of slider when window is resized in vault as well as NOT repacking the ads when leaving inspection. Double check if that's what you meant.
